applications/StructuralMechanicsApplication/custom_utilities/perturb_geometry_base_utility.cpp

```diff
@@ -30,7 +31,7 @@ PerturbGeometryBaseUtility::PerturbGeometryBaseUtility( ModelPart& rInitialModel
         mMaximalDisplacement(Settings["max_displacement"].GetDouble())
     {
         KRATOS_TRY
-        MortarUtilities::ComputeNodesMeanNormalModelPart( mrInitialModelPart, false );
+        NormalCalculationUtils().CalculateUnitNormals<Element>(mrInitialModelPart, true);
```

Comment:
@manuelmessmer can you take a look here?

Reply:
Yes, I will

Reply:
![image](https://user-images.githubusercontent.com/25484702/94408081-9bffd180-0174-11eb-8ee8-85fd289013f5.png)

Reply:
And I'd have gotten away with it, too, if it weren't for those meddling kids!

Reply:
Mysterious... ;)
If I got it right. The `NormalCalculationUtils().CalculateUnitNormals<Element>` computes the normals for each element, but also assigns the mean normal to all nodes, correct?
Then everything should be fine, as the utility only requires the normal direction for each node. (Each node is perturbed along its normal direction)
https://github.com/KratosMultiphysics/Kratos/blob/4a902bbb8f6090ea83c18c0faa7fc6c5ecd2708e/applications/StructuralMechanicsApplication/custom_utilities/perturb_geometry_base_utility.cpp#L91